requirements.txt

```diff
@@ -3,8 +3,8 @@ pandas>=0.23.0
 tqdm>=4.19.2
```

Comment:
Do we use tqdm directly in this library?

Reply:
It doesn't seem we do. Should I remove?

Reply:
Yeah let's remove

Reply:
removed